test/functions_test.rb

```diff
@@ -141,8 +141,9 @@ def sass_return_path(path)
         Script::String.new("#{path.value}", :string)
       end
 
-      def optional_arguments(path, optional = "bar")
-        Script::String.new("#{path.value}/#{optional}", :string)
+      def optional_arguments(path, optional = nil)
+        optional ||= Script::String.new("bar")
```

Comment:
@bolandrm that is an excellent question. I’ll test it out on Monday and follow up.

CC @rafaelfranca since you should know about this stuff too.